@here/harp-examples/src/bounds-generation.ts

```diff
@@ -0,0 +1,186 @@
+import { OmvDataSource } from "@here/harp-omv-datasource";
+
+import { apikey } from "../config";
+
+export namespace BoundsExample {
+    // Create a new MapView for the HTMLCanvasElement of the given id.
+    function initializeMapView(id: string): MapView {
+        const canvas = document.getElementById(id) as HTMLCanvasElement;
+
+        // Look at BERLIN
+        const BERLIN = new GeoCoordinates(52.5186234, 13.373993);
+        const map = new MapView({
+            canvas,
+            projection: mercatorProjection,
+            tileWrappingEnabled: false,
+            theme: {
+                extends: "resources/berlin_tilezen_night_reduced.json",
+                styles: {
+                    geojson: [
+                        {
+                            when: ["all", ["==", ["geometry-type"], "Polygon"]],
+                            technique: "fill",
+                            renderOrder: 10003,
+                            attr: {
+                                color: ["get", "color"], // "#77ccff",
+                                opacity: 0.5,
+                                lineWidth: 1,
+                                lineColor: "#ff0000",
+                                enabled: true
+                            }
+                        },
+                        {
+                            when: "$geometryType == 'point'",
+                            technique: "circles",
+                            renderOrder: 10004,
+                            attr: {
+                                color: "#f0f",
+                                size: 10
+                            }
+                        }
+                    ]
+                }
+            },
+            target: BERLIN,
+            zoomLevel: 8,
+            tilt: 45,
+            heading: -80
+        });
+        map.renderLabels = false;
+
+        CopyrightElementHandler.install("copyrightNotice", map);
+
+        const mapControls = new MapControls(map);
+        mapControls.maxTiltAngle = 180;
+
+        const ui = new MapControlsUI(mapControls, { zoomLevel: "input" });
+        canvas.parentElement!.appendChild(ui.domElement);
+
+        map.resize(window.innerWidth, window.innerHeight);
+
+        window.addEventListener("resize", () => {
+            map.resize(window.innerWidth, window.innerHeight);
+        });
+
+        addOmvDataSource(map);
+        const featureList: MapViewFeature[] = []; //createFeatureList();
+        let featuresDataSource: FeaturesDataSource | undefined;
+        //@ts-ignore
+        featuresDataSource = addFeaturesDataSource(map, featureList);
+        let viewpoly: MapViewPolygonFeature;
+        let cornerpoints: MapViewMultiPointFeature;
+        const boundsGenerator = new BoundsGenerator(
+            map.camera,
+            map.projection,
+            map.tileWrappingEnabled
+        );
+
+        window.addEventListener("keyup", event => {
```

Comment:
please add some instructions how to use this example, otherwise without looking at the examples code, I don't know how to use it